### DISCUSSION TITLE
What should be the generation speed on RTX 3060?

### DISCUSSION THREAD

**Discussion by ray2022** [oobabooga/text-generation-webui#506] [General]
When using Llama 13b 4bit I get around 2.5 tokens/s (tested in the notebook mode for 200 tokens with just a one token when starting).
It's strange because I get better speed on CPU using llama.cpp and 13b 4bit. Also I saw [tomshardware benchmark](https://www.tomshardware.com/news/running-your-own-chatbot-on-a-single-gpu) where they got 19.5 tokens/s on the same repo and card. What results do you have?

**Comment by mindcz** [comment]
same for me here on rtx3070... speed is really slow(
